katharsis-jpa/pom.xml

```diff
@@ -0,0 +1,241 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>io.katharsis</groupId>
+		<artifactId>katharsis-parent</artifactId>
+		<version>2.6.0-SNAPSHOT</version>
+		<relativePath>../katharsis-parent</relativePath>
+	</parent>
+
+	<artifactId>katharsis-jpa</artifactId>
+	<packaging>bundle</packaging>
+	<name>katharsis-jpa</name>
+
+	<properties>
+		<h2.version>1.4.187</h2.version>
+		<hibernate.version>5.1.0.Final</hibernate.version>
+		<hibernate.jpa.version>1.0.0.Final</hibernate.jpa.version>
+		<commons.beanutils.version>1.9.2</commons.beanutils.version>
+		<spring.version>4.3.1.RELEASE</spring.version>
+		<javaee-api.version>7.0</javaee-api.version>
+		<jersey.version>2.17</jersey.version>
+		<javax.ws.rs-api.version>2.0.1</javax.ws.rs-api.version>
+		<querydsl.version>4.1.3</querydsl.version>
+	</properties>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.jacoco</groupId>
+				<artifactId>jacoco-maven-plugin</artifactId>
+			</plugin>
+			<plugin>
+				<groupId>org.eluder.coveralls</groupId>
+				<artifactId>coveralls-maven-plugin</artifactId>
+			</plugin>
+			<plugin>
+				<groupId>com.github.danielflower.mavenplugins</groupId>
+				<artifactId>gitlog-maven-plugin</artifactId>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.felix</groupId>
+				<artifactId>maven-bundle-plugin</artifactId>
+				<configuration>
+					<instructions>
+						<Export-Package>io.katharsis.jpa.*,io.katharsis.jpa.query.*,io.katharsis.jpa.querydsl.*,io.katharsis.jpa.criteria.*</Export-Package>
+					</instructions>
+				</configuration>
+			</plugin>
+
+			<plugin>
+				<groupId>com.mysema.maven</groupId>
+				<artifactId>apt-maven-plugin</artifactId>
+				<version>1.1.1</version>
+				<executions>
+					<execution>
+						<phase>generate-test-sources</phase>
+						<goals>
+							<goal>test-process</goal>
+						</goals>
+						<configuration>
+							<outputDirectory>target/generated-test-sources</outputDirectory>
+							<processor>com.querydsl.apt.jpa.JPAAnnotationProcessor</processor>
+						</configuration>
+					</execution>
+				</executions>
+				<dependencies>
+				<dependency>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this line start at column 11. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)